dotCMS/src/main/java/com/dotmarketing/business/PermissionBitFactoryImpl.java

```diff
@@ -1258,14 +1258,31 @@ private PersistResult persistPermission(Permission p) throws DotDataException {
 	@Override
     @Deprecated
```

Comment:
if it is deprecated please doc, which one is the new method or the suggestion to replace

Reply:
done https://github.com/dotCMS/core/pull/21780/commits/c2ceed6de728d4fdb5001ed39d2f3760736a42b5#diff-e748015d1e1cc6e815fbfafdac7576623968742461c5525522cf3f9db77f8c29R1255